src/app/collection/config/tab/attribute-list/attribute-list.component.ts

```diff
@@ -71,12 +72,11 @@ export class AttributeListComponent {
               private notificationService: NotificationService) {
   }
 
-  private emptyAttribute(): ConfiguredAttribute {
+  private emptyAttribute(): Attribute {
```

Comment:
We should rather use `AttributeModel` class so we do not deal with DTO objects outside of the services and store.